suites/pacific/cephadm/tier-1_5-1_service-apply-spec.yaml

```diff
@@ -348,3 +348,11 @@ tests:
         check_status: True
         commands:
           - "ceph status"
+  - test:
+      name: test_cephadm_pull
+      desc: Check the latest ceph version using cephadm pull
+      polarion-id: CEPH-83573787
+      module: test_cephadm_pull.py
+      config:
+        url : http://magna002.ceph.redhat.com/cephci-jenkins/latest-rhceph-container-info/release.yaml
```

Comment:
This test case forces the test suite to be tagged as `psi-only`. Is this something that all are in agreement.

I noticed that this suite is not being executed as part of the pipeline... is this something that was done intentionally?

Reply:
Can we avoid hard-coding the URL but maintain a list of locations based on `cloud-type` making the test suite generic?

Reply:
@psathyan 
Thanks for the review comments.

As discussed
- Moved test case to test suite "tier-1_service_apply_spec.yaml" which is the part of pipeline.
- Removed hard cored "url" and added new method "fetch_image_tag" in util.
- Now comparing the image from podman image list(Executed after cephadmpull) with image from recipe file.
- Not comparing ceph version.